skelebot/systems/scaffolding/scaffolder.py

```diff
@@ -47,18 +64,33 @@ def scaffold(self):
 
         # Configure Template Variables
         self.variables["name"] = name
-        self.variables["name_simple"] = name.lower().replace(" ", "_")
+        self.variables["name_simple"] = name.lower().replace(" ", "_").replace("-", "_")
         self.variables["description"] = description
         self.variables["maintainer"] = maintainer
         self.variables["contact"] = contact
         self.variables["language"] = language
 
         # Load the Template Config
-        template = promptUser("Select a TEMPLATE", options=list(TEMPLATES[language].keys()))
-        template_name = TEMPLATES[language][template]
-        template_path = TEMPLATE_PATH.format(name=template_name)
-        template_path = os.path.join(os.path.dirname(__file__), template_path)
-        template = self.__load_template(template_path) 
+        options = list(TEMPLATES[language].keys())
+        options.append("github:{owner}:{repo}:{branch}")
```

Comment:
I was a little confused by the workflow here. I pulled the branch locally to test.
It was not immediately obvious that you have to provide `owner` and `repo` information but `branch` is optional.
We either need to put that in the docs or make 3 separate prompts for each information. The second option seems nicer to me but more work to implement.

We also may make want to think about how to generalize it to other hosts (gitlab, bitbucket, etc.)generalize

Reply:
Yes! I was thinking to make 3 separate prompts for each information as well.
And yes, it would be nice to include other hosts as well, but would require some special code for each to get the files I'm sure. (unless we just want to switch to use `git clone`)

Reply:
git clone would add a dependency: what if the user does not have git installed?
I bet other hosts have some canonical url format to directly download files. But you're right, this would be something for down the line.